src/main/java/at/ac/tuwien/kr/alpha/solver/learning/GroundConflictNoGoodLearner.java

```diff
@@ -75,33 +79,36 @@ public int computeConflictFreeBackjumpingLevel(NoGood violatedNoGood) {
 
 		public final NoGood learnedNoGood;
 		public final int backjumpLevel;
-		public final boolean clearLastChoiceAfterBackjump;
 		public final Collection<Integer> resolutionAtoms;
+		public final int lbd;
 
 		private ConflictAnalysisResult() {
 			learnedNoGood = null;
 			backjumpLevel = -1;
-			clearLastChoiceAfterBackjump = false;
 			resolutionAtoms = null;
+			lbd = -1;
```

Comment:
`-1` is a magic number used at several places, so it should be moved to a constant with appropriate JavaDoc